examples/graphql_composable/graphql_composable.info.yml

```diff
@@ -6,3 +6,4 @@ core: 8.x
 dependencies:
   - graphql:graphql
   - node:node
+core_version_requirement: 8.x
```

Comment:
should be `core_version_requirement: ^8 || ^9` like in the main graphql.info.yml file